.gitattributes

```diff
@@ -1,4 +1,7 @@
 *.go text eol=lf
+**/testdata/**/*.snap text eol=lf
+**/testdata/**/*.md text eol=lf
+*.txt text eol=lf
```

Comment:
Do we still need this? I notice that `*.snap` and `*.txt` is noted below

Reply:
My bad. Somehow I keep bringing this change with all my PRs LoL.  I need to fix this.